features/accelerometer.yml

```diff
@@ -0,0 +1,7 @@
+name: Accelerometer
+description: The `Accelerometer` and `LinearAccelerationSensor` APIs read the acceleration applied to a device in three dimensions, either including the effect of gravity or not, respectively.
+spec: https://w3c.github.io/accelerometer/#accelerometer-interface
+group: sensors
+compat_features:
+  - api.Accelerometer
```

Comment:
Here and elsewhere, can you include the interface members too? My starting point for a list would be `npm run traverse | grep Accelerometer` and similar.

Reply:
The constructor is missing. I'll add it.